lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -1732,10 +1732,9 @@ LogicalResult FIRRTLLowering::visitDecl(MemOp op) {
     auto portName = op.getPortName(i).getValue();
     auto portKind = op.getPortKind(i);
 
-    auto &portKindNum =
-        portKind == MemOp::PortKind::Read
-            ? readCount
-            : portKind == MemOp::PortKind::Write ? writeCount : readwriteCount;
+    auto &portKindNum = portKind == MemOp::PortKind::Read    ? readCount
+                        : portKind == MemOp::PortKind::Write ? writeCount
+                                                             : readwriteCount;
 
```

Comment:
Did clang-format do this, or was this by hand?  Please stick with whatever clang format is producing by default.

Reply:
Right, this is done by clang-format. I believe `DShlPrimOp` and `DShlwPrimOp::getResultType` works well.

Reply:
clang-9 and clang-11 differ on how they format this.  I have clang-9 installed and use it's formatter rather than more current ones.